<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ch2.qrc">
    <normaloff>:/OCR.png</normaloff>:/OCR.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="IP_lineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>200</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Port_lineEdit">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>200</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>60</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-weight:700;&quot;&gt;客户端&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>200</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>PLC地址：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>200</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>PLC端口：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Connect_pushButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>430</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>连接</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DisConnect_pushButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>430</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>断开连接</string>
    </property>
   </widget>
  </widget>
  <action name="action_Open">
   <property name="icon">
    <iconset theme="打开" resource="ch2.qrc">
     <normaloff>:/OCR.png</normaloff>:/OCR.png</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>打开文件</string>
   </property>
  </action>
  <action name="action_New">
   <property name="icon">
    <iconset theme="创建" resource="ch2.qrc">
     <normaloff>:/OCR.png</normaloff>:/OCR.png</iconset>
   </property>
   <property name="text">
    <string>创建</string>
   </property>
   <property name="toolTip">
    <string>创建文件</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset theme="保存" resource="ch2.qrc">
     <normaloff>:/OCR.png</normaloff>:/OCR.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>保存文件</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="ch2.qrc"/>
 </resources>
 <connections/>
</ui>
